app/models/category.rb

```diff
@@ -15,4 +15,8 @@ def to_json
   def self.to_json
     includes(:items).order(:description).all.map(&:to_json).to_json
   end
+
+  def value
+    items.all.map(&:value).inject(:+)
```

Comment:
How about `Item.all.sum(:value)` instead. This will do a single sum query:
```
2.2.4 :004 > Item.all.sum(:value).to_f
   (0.9ms)  SELECT SUM("items"."value") FROM "items"
 => 325.06
```